msmbuilder/featurizer/featurizer.py

```diff
@@ -884,6 +884,55 @@ def __init__(self, mode='residue', **kwargs):
     def partial_transform(self, traj):
         return md.shrake_rupley(traj, mode=self.mode, **self.kwargs)
 
+    def describe_features(self, traj):
+        """Return a list of dictionaries describing the SASA features.
+
+        Parameters
+        ----------
+        traj : mdtraj.Trajectory
+            The trajectory to describe
+
+        Returns
+        -------
+        feature_descs : list of dict
+            Dictionary describing each feature with the following information
+            about the atoms participating in each SASA feature
+                - resnames: names of residues
+                - atominds: atom index or atom indices in mode="residue"
+                - resseqs: residue ids (not necessarily 0-indexed)
+                - resids: unique residue ids (0-indexed)
+                - featurizer: SASA
+                - featuregroup: atom or residue
+        """
+
+        feature_descs = []
+        _, mapping = md.geometry.sasa.shrake_rupley(traj, mode=self.mode, get_mapping=True)
+        top = traj.topology
+
+        if self.mode == "residue":
+            resids = np.unique(mapping)
+            resseqs = [top.residue(ri).resSeq for ri in resids]
+            resnames = [top.residue(ri).name for ri in resids]
+            # Not sure if there's a better way to get the indices of atoms in a residue
+            atoms_in_res = [res.atoms for res in top.residues]
```

Comment:
This looks good to me and i cant think of a fancy list comprehensive way around this. Maybe @mpharrigan knows.